beacon-chain/verification/result.go

```diff
@@ -0,0 +1,64 @@
+package verification
+
+// represents a verification requirement.
+// Requirement represents a validation check that needs to pass in order for a Verified form a consensus type to be issued.
+type Requirement int
+
+// results collects positive verification results.
+// This bitmap can be used to test which verifications have been successfully completed in order to
+// decide whether it is safe to issue a "Verified" type variant.
+type results struct {
+	done map[Requirement]error
+	reqs []Requirement
+}
```

Comment:
This type has some test coverage indirectly via `blob_test.go` but it probably should have it's own tests.